pkgs/applications/graphics/eddy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ python37Packages
+, fetchFromGitHub
+, lib
+, jre
+, qt5
+}:
+
+python37Packages.buildPythonApplication rec {
+  pname = "eddy";
+  version = "v1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "obdasystems";
+    repo = pname;
+    rev = version;
+    sha256 = "12j77bbva5py9bd57c80cmjvf8vll40h19n81h16lvv2r2r7jynh";
+  };
+
+  propagatedBuildInputs = [
+    qt5.full
```

Comment:
Do we really need full?

Reply:
What can I use instead?

Reply:
base?

Reply:
Everything works fine with qt5.full. With qt5.qtbase, there's the following error:

```
Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
```

After setting, `dontWrapQtApps` to `false`, I then get the following error:

```
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted (core dumped)
```

Do you know how to solve this. I'm not very familiar with Qt.

Reply:
See https://nixos.wiki/wiki/Qt

You want to use WrapQtApps here.

Reply:
Alright, it works. I now use `qt5.qtbase` and `wrapQtAppsHook` dependencies with `wrapQtApp`.